src/test/java/org/openstreetmap/atlas/checks/validation/intersections/SelfIntersectingPolyLineCheckTest.java

```diff
@@ -119,4 +121,11 @@ public void testInvalidAreaBuildTag()
         this.verifier.actual(this.setup.getInvalidAreaBuildingTag(), check);
         this.verifier.verifyExpectedSize(1);
     }
+
+    @Test
+    public void testInlineConfig()
```

Comment:
Are you testing the inline config here, or the minimum highway type, or both? If so, you should indicate that in the test name

Reply:
I missed to update this when I added the `tag.filter` back. This is testing the minimum highway type.